engine/gamesys/src/gamesys/components/comp_collection_proxy.cpp

```diff
@@ -78,14 +78,21 @@ namespace dmGameSystem
         uint32_t                        m_DelayedEnable : 1;
         uint32_t                        m_Unloaded : 1;
         uint32_t                        m_AddedToUpdate : 1;
+        uint32_t                        m_Loading : 1;
 
         dmResource::HPreloader          m_Preloader;
         dmMessage::URL                  m_LoadSender, m_LoadReceiver;
 
         ProxyLoadCallback               m_Callback;
         void*                           m_CallbackCtx;
+        char*                           m_CollectionResPath;  // set from script as an override
     };
```

Comment:
This struct could be cleaned up a bit. We usually try to sort by member size, and we never put members on the same line like the m_LoadSender, m_LoadReceiver.